app/handlers/GuaranteeBalanceResponseHandler.scala

```diff
@@ -152,7 +153,7 @@ sealed trait GuaranteeBalanceResponseHandler extends Logging {
 
     auditService.audit(
       SuccessfulBalanceAuditModel.build(
-        request.userAnswers.get(EoriNumberPage).getOrElse("-"),
+        eoriNumber,
```

Comment:
Also very minor but can we rename the `build` methods to `apply` to be more scala-y 😄